bin/resources/GameIndex.yaml

```diff
@@ -17972,7 +17972,7 @@ SLES-55196:
   roundModes:
     vuRoundMode: 2  # Fixes very minor lines appearing at certain points during the game.
 SLES-55197:
-  name: "Dancing Stage SuperNOVA 2"
+  name: "Dancing Stage SuperNOVA2"
```

Comment:
on the box this looks separated to me https://www.picclickimg.com/d/l400/pict/164839544556_/Dancing-Stage-Supernova-2-Sony-Playstation-2-Ps2.jpg

Reply:
You're right, I just checked the official Konami pages for that and it's stylized with a space there. I'll revert those two.

Reply:
After your comment I did a double check and noticed that the Japanese version of SuperNOVA 2 wasn't on the list so I went over the Japanese PS2 releases as a whole and was able to add SLPM-55090 (DDR X) and SLPM-66930 (SuperNOVA 2). The other versions in the GameDB have `compat: 5` but I am not really sure what that is or how to test so I didn't it. If there's something specific I can test to check if compat 5 is required then please let me know.

https://www.jp.playstation.com/software/title/slpm55090.html
https://www.jp.playstation.com/software/title/slpm66930.html